packages/serverpod/lib/src/server/server.dart

```diff
@@ -237,8 +243,16 @@ class Server {
         return;
       }
       webSocket.pingInterval = const Duration(seconds: 30);
```

Comment:
Question: Should we expose this setting for the end user? If we do we give a little more power for people to either reduce network traffic or increase confidence in that clients are alive, might be relevant depending on what you are building.

Reply:
I agree with this as well! Would be nice to have handles for these things :)